ai_diffusion/workflow.py

```diff
@@ -179,6 +179,20 @@ def _sampler_params(
     return params
 
 
+def _apply_strength(strength: float, params: dict, fixed_steps: bool = False) -> dict[str, Any]:
+    steps = params['steps']
+    start_at_step = round(steps*(1-strength))
```

Comment:
I'd still like to see this clamped somewhere so that at least one step runs
eg. 10 steps @ 4% strength should not result in `steps==start_at_step` an no sampling happening

Reply:
Done, and the calculation code is now somewhat more readable.